include/tfm/tfm_ioctl_api.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef TFM_IOCTL_API_H__
+#define TFM_IOCTL_API_H__
+
+#include <limits.h>
+#include <stdint.h>
+#include <tfm_api.h>
+#include <tfm_platform_api.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum tfm_platform_ioctl_reqest_types_t {
+    TFM_PLATFORM_IOCTL_READ_SERVICE
+};
+
+/*!
+ * \struct tfm_read_service_args_t
```

Comment:
@b-gent  should these docs be in ncs or tfm style?

Reply:
The issue is that this file is built by the tfm built system, but it is hosted in the nrf repository.

Reply:
Interesting case.
'Built by the tfm built system' means they are originating from the TF-M repo and not created by us?

Reply:
No, they are created by us. But they are only built by the TF-M repo build system.